src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -576,9 +576,11 @@ namespace wallet_rpc
     struct request
     {
       std::string unsigned_txset;
+      std::string multisig_txset;
```

Comment:
You're adding something, so please bump the minor version number at the top of the file.

Reply:
bumped it to 8